### PR TITLE
Release/1.0 final changes

### PR DESCRIPTION
**Pull request by EllieShivers**

### CODE REVIEW

File: src/app/components/form-viewer/form-viewer.component.ts [hunk 1]
```diff
@@ -48,25 +55,36 @@ export class FormViewerComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit(): void {
+    // Results Handling
+    // let results$ = this.refreshTrigger$.pipe(
```
Comment (plamentassev) [inline src/app/components/form-viewer/form-viewer.component.ts:59]: delete unused code

File: src/app/services/rc-api-interface/rc-api-interface.service.ts [hunk 2]
```diff
@@ -197,7 +212,7 @@ export class RcApiInterfaceService {
             results[linkId].evidence = [... new Set(results[linkId].evidence)];
           }
         });
-        console.log(results);
+        // console.log(results);
```
Comment (plamentassev) [inline src/app/services/rc-api-interface/rc-api-interface.service.ts:215]: delete

File: src/app/components/form-viewer/patient-details/patient-details.component.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  identifierLabel: string = "Medical Record Number";
+  identifierSystem: string | undefined = undefined;
+  identifierValue: string = "Unknown";
+
+  constructor(public rcApiInterface: RcApiInterfaceService) {
+
+  }
+
+  ngOnInit() {
```
Comment (plamentassev) [inline src/app/components/form-viewer/patient-details/patient-details.component.ts:22]: Remove onInit (not used)

File: src/app/components/form-viewer/patient-details/patient-details.component.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+
+
```
Comment (plamentassev) [inline src/app/components/form-viewer/patient-details/patient-details.component.scss:2]: delete commented code (its just dead code I think)

File: src/app/components/form-viewer/form-viewer.component.ts (hunk 1 above)
Comment (EllieShivers) [reply src/app/components/form-viewer/form-viewer.component.ts:59]: Left in as placeholder for later implementation.

**Pull request merged**